.github/workflows/wasm-benchmarks.yml

```diff
@@ -22,6 +22,15 @@ jobs:
       - name: Checkout PR branch
         uses: actions/checkout@v4
 
+      - uses: dtolnay/rust-toolchain@nightly
+        with:
+          toolchain: nightly-2024-01-25
```

Comment:
Is there any way we could have it specified in a single place? Is it possible to pass a path to `rust-toolchain.toml` file perhaps?

Reply:
https://github.com/dtolnay/rust-toolchain/issues/77 🙄